pkg/messagebroker/models.go

```diff
@@ -8,7 +10,9 @@ type CreateTopicRequest struct {
 
 // DeleteTopicRequest ...
 type DeleteTopicRequest struct {
-	Name string
+	Name         string
+	Force        bool
+	NonPartioned bool
```

Comment:
Add a comment saying `Force` and `NonPartioned ` are only requited for pulsar and ignored for kafka